source/_patterns/00-atoms/components/tweet.mustache

```diff
@@ -0,0 +1,13 @@
+<div class="tweet">
+  <blockquote class="twitter-tweet"{{#hideConversation}} data-conversation="none"{{/hideConversation}}{{#hideCards}} data-cards="hidden"{{/hideCards}}>
+    {{#text}}
+      {{> atoms-paragraph }}
+    {{/text}}
+    &mdash; {{accountLabel}} (@{{accountId}}) <a href="{{url}}">
+    {{#date}}
+      {{>atoms-date}}
+    {{/date}}
+    </a>
+  </blockquote>
+  <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Wasn't sure which was the favoured approach to introduce this library, so it was introduced like this to get things working quickly.

Reply:
👍 For the actual implementation, it might be worth looking at [MathJax is handled].(https://github.com/elifesciences/pattern-library/blob/develop/assets/js/components/Math.js)